exasol_script_languages_developer_sandbox/templates/vm_bucket_cloudformation.jinja.yaml

```diff
@@ -46,6 +44,65 @@ Resources:
                   - ec2:Describe*
                 Resource:
                   - "*"
+  S3CDNBucketPolicy:
+    Metadata:
+      Comment: 'Bucket policy to allow cloudfront to access the images'
+    Properties:
+      Bucket: !Ref {{bucket_name}}
+      PolicyDocument:
+        Statement:
+          - Action:
+              - 's3:GetObject'
+            Effect: 'Allow'
+            Principal:
+              CanonicalUser: !GetAtt CfOriginAccessIdentity.S3CanonicalUserId
+            Resource:
+              - !Sub "arn:aws:s3:::${ {{bucket_name}} }/slc_developer_sandbox/*"
+    Type: 'AWS::S3::BucketPolicy'
+
+  CfDistribution:
+    Type: 'AWS::CloudFront::Distribution'
+    Properties:
+      DistributionConfig:
+        Comment: 'A simple distribution with an S3 origin'
+        DefaultCacheBehavior:
+          AllowedMethods:
+            - 'HEAD'
+            - 'GET'
+          CachedMethods:
+            - 'HEAD'
+            - 'GET'
+          Compress: false
+          DefaultTTL: 86400
+          ForwardedValues:
+            Cookies:
+              Forward: 'none'
+            Headers:
+              - 'Origin'
+            QueryString: false
+          MaxTTL: 31536000
+          MinTTL: 86400
+          TargetOriginId: !Sub 's3-origin-${ {{bucket_name}} }'
+          ViewerProtocolPolicy: 'redirect-to-https'
+        DefaultRootObject: 'index.html'
+        Enabled: true
+        HttpVersion: 'http1.1'
+        IPV6Enabled: true
+        Origins:
+          - DomainName: !GetAtt {{bucket_name}}.DomainName
+            Id: !Sub 's3-origin-${ {{bucket_name}} }'
+            OriginPath: ''
+            S3OriginConfig:
+              OriginAccessIdentity: !Sub 'origin-access-identity/cloudfront/${CfOriginAccessIdentity}'
+        PriceClass: 'PriceClass_All'
+
+  CfOriginAccessIdentity:
```

Comment:
OriginAccessIdentity is legacy and not recommended anymore. Now you should use OriginAccessControl


https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/private-content-restricting-access-to-s3.html

Reply:
Hm, interesting. I got the template from the AWS docu:
https://aws.amazon.com/blogs/networking-and-content-delivery/amazon-s3-amazon-cloudfront-a-match-made-in-the-cloud/

(linked in this blog https://aws.amazon.com/blogs/networking-and-content-delivery/amazon-s3-amazon-cloudfront-a-match-made-in-the-cloud/)